staging/src/k8s.io/client-go/util/certificate/certificate_manager.go

```diff
@@ -215,11 +232,30 @@ func (m *manager) Start() {
 
 	glog.V(2).Infof("Certificate rotation is enabled.")
 
+	lastTemplate := m.getTemplate()
+	if m.dynamicTemplate && lastTemplate == nil {
+		// wait for our first template
+		wait.PollImmediateInfinite(time.Second, func() (bool, error) {
+			lastTemplate = m.getTemplate()
+			if lastTemplate == nil {
+				glog.Infof("Waiting for certificate template")
```

Comment:
Why not v(2)?  Because it hangs?  If so, might want a slightly longer message.

Reply:
Yeah, wanted visibility